scholarx/archive/functions.js

```diff
@@ -263,40 +248,83 @@ function filterByYear() {
     const selectedIndustries = selectedIndustriesData();
     const selectedUniversities = selectedUniversitiesData();
     //helps to achieve mentees page cofunctionality with cohort filters 
-    if(document.getElementById("selection").value === "mentees"){
-        if(selectedUniversities.length == 0 && selectedYears.length == 0){
+    if (document.getElementById("selection").value === "mentees") {
+        if (selectedUniversities.length == 0 && selectedYears.length == 0) {
             renderAllProfiles()
             return
         }
         if (selectedUniversities.length == 0) {
-           filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
-           filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
-           renderProfiles(filteredMentors, filteredMentees);
-        } else{
+            filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
+            filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
+            renderProfiles(filteredMentors, filteredMentees);
+        } else {
             filterByUniversity("")
         }
         return;
     }//mentors page cofunctionality
-    if(selectedIndustries.length == 0 && selectedYears.length == 0){
+    if (selectedIndustries.length == 0 && selectedYears.length == 0) {
         renderAllProfiles()
         return
     }
     if (selectedIndustries.length == 0) {
-       filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
-       filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
-       renderProfiles(filteredMentors, filteredMentees);
-    } else{
+        filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
+        filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
+        renderProfiles(filteredMentors, filteredMentees);
+    } else {
         filterByIndustry("")
     }
 }
-function openMentorProfile(index){
+//mentor mentee selection
+$(document).ready(function () {
+    $('#selection').on('change', function () {
```

Comment:
```suggestion
    function handlePageTransition() { ...
```
Instead of **change** we can create a function like this and use **onClick** attribute in **input** tags

Reply:
Ok that's also done but that mentee page is not hiding I make to keep the mentors checked when it is loaded again and again but mentee page?